src/state/apis/zapper/zapperApi.ts

```diff
@@ -418,8 +433,15 @@ export const zapper = createApi({
                   // This is our best bet until we bring in the concept of an "DefiType.GenericOpportunity"
                   const defiType = DefiType.Staking
 
-                  // Actually defined because we pass networks in the query params
-                  const assetId = zapperAssetToMaybeAssetId(asset)
+                  const topLevelAsset = (() => {
```

Comment:
this bit looks very similar to `stakedAmountCryptoBaseUnitAccessor`, is `balanceRaw` check not applicable here? can it use the same value as above?

Reply:
Yeah the fact those are two different constant is on purpose here, we need the `balanceRaw` check for the former (i.e for the staked amount, is there a non-null, non-zero `balanceRaw` amount,  not for the latter